custom/succeed/models.py

```diff
@@ -107,4 +107,43 @@ class UCLAPatientFluff(fluff.IndicatorDocument):
 
     emitter = RandomizationDate()
 
+
+def get_full_name(case):
+    try:
+        return CommCareCase.get(case['indices'][0]['referenced_id'])['full_name']
+    except AttributeError:
+        return None
+
+
+class Emitter(fluff.Calculator):
+
+    @fluff.null_emitter
+    def date(self, case):
+        yield None
+
+
+class UCLATaskActivityFluff(fluff.IndicatorDocument):
```

Comment:
It looks like this could be very close to writeable with a `userreports.models.CustomDataSourceConfiguration` (would just require one small custom function for the `full_name`). What do you think about converting it? We are trying to shift to those whenever possible to have a standardized infrastructure and not introduce new overhead/pillows with every new table.

If this is urgent or sounds too hard it doesn't need to block - looks like it could be quick and easy though